### ISSUE TITLE
Plugin for Mozilla&other maybe  under Mozilla's/FF/addons

### ISSUE THREAD

**Issue by blackcrack** [owncloud/mozilla_sync#132]
Hy, anyone there in github and OC-Programmers can maby create an FF-Addon, this is more better as the internal one, because, an App can you get more Configurations possibility's and, we are can download it and can linking from any web pages.., so, maybe an easy FF Addon .. ?

best regards
Blacky

**Comment by Linux-Ham** [comment]
I agree, it getting bit annoying having to reset sync to owncloud server cause of FF update or it just wants to have a spack and reset the sync to Mozilla defaults, so it be good to have an addon that just took care of it.
